fretboard-generator.js

```diff
@@ -12,11 +12,15 @@ var makerjs = require('makerjs');
  * @param {number} frets - How many frets to calculate for.
  * @return {[number]} A list of positions in the same units as the scale length was provided.
  */
-function generateFretPositions(scaleLength, frets) {
+function generateFretPositions(scaleLength, frets, bridgeLocation) {
```

Comment:
Can you please also update the document comments above to include the new parameter?